email_tracking/app.rb

```diff
@@ -0,0 +1,62 @@
+require 'sinatra/base'
+require 'easypost'
+require 'sendgrid-ruby'
+require 'tilt/erb'
+require 'pry'
+require 'dotenv'
+
+include SendGrid
+
+class App < Sinatra::Base
+  set :show_exceptions, :after_handler
+
+  configure :development, :test do
+    Dotenv.load
+  end
+
+  configure do
+    EasyPost.api_key = ENV['EASYPOST_API_KEY']
+    set :sendgrid_key, SendGrid::API.new(api_key: ENV['SENDGRID_API_KEY'])
+  end
+
+
+ post '/easypost-webhook' do
+    sendgrid = settings.sendgrid_key
+    request.body.rewind
+    request_string = request.body.read.to_s
+    response = JSON.parse(request_string)
+
+    if response['object'] == 'Event' && response['description'] == 'tracker.updated'
+      event = EasyPost::Event.receive(request_string)
+      tracker = event.result
+
+      @message = "Hey, this is FunCompany."
```

Comment:
sorry.. I will remove it... I had added it for some reason before.. forgot to remove it..